### DISCUSSION TITLE
Sicurezza Mini-PC

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#2872] [Hardware & Firmware]
Ciao a tutti,

grazie al vostro sito e alle vostre guide mi si è aperto un nuovo mondo :) Al momento stra preso da tutto ciò.

Recentemente quindo ho acquistato un Mini PC e ho installato HA Core su Debian. 

Ho sempre e solo lavorato in ambiente Windows e Debian per me è un mondo completamente nuovo e proprio su quest'ultimo ho alcuni dubbi..probabilmente da principiante :) 

Uno di questi riguarda la sicurezza. Su Debian non ho configurato alcun antivirus, firewall etc cose che su Windows di solito fai quasi per primo. 
Cose ne pensate? Devo fare qualcosa a riguardo oppure dato che il Mini PC è dedicato solo per HA i rischi sono minimi. Pensando anche a quando abiliterò l'accesso remoto..

Grazie per gli eventuali consigli!

**Comment by indomus** [comment]
Benvenuto!
Partiamo dal presupposto che la piattaforma di attacco di Linux Debian è ridottissima di base, e tu non hai fatto altro che installare HA (su Docker, probabile) e attivare SSH. Inoltre non dovresti aver esposto il Mini PC a Internet... quindi, ciò detto, dovresti di base essere relativamente al sicuro. Di certo non c'è bisogno di installare antivirus. Buono cmq che tu ti ponga certe domande.

**Reply by Asky82** [reply]
grazie 1000 per il feedback!

a dire il vero dal MiniPC posso navigare su internet, quindi immagino sia esposto se si intende questo.  Ma non dovrebbe esserlo in ogni modo anche solo per scaricare updates , per far funzionare i dispositivi cloud o per l'accesso remoto?

**Reply by Scialla** [reply]
esposto a internet significa che da internet vedi il tuo miniPC, non il contrario..

**Reply by Asky82** [reply]
Giusto :) 

Ma nel caso volessi collegarmi da remoto a HA(seguendo le procedure sul sito) ci sarebbe qualcosa da tenere in considerazione in termini di sicurezza?

**Reply by indomus** [reply]
se usi un [tunnel Cloudflare](https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/), sostanzialmente devi solo ricordarti di mettere una password robusta e il 2FA.
